ui/src/pages/backoffice/Loan/LoanDetails/components/LoanMetadata/LoanMetadata.js

```diff
@@ -76,16 +78,22 @@ export default class LoanMetadata extends Component {
     );
   }
 
+  getLinkToItem = itemPid => {
```

Comment:
there is another usage below, could it be a component (next PR)? I'm trying to avoid the methods in favour of the components, it's easier to give them a common look

Reply:
Oh yes, sure! I will take care of this in the next PR!